src/components/Connection/Desktop/CreateAccount.vue

```diff
@@ -0,0 +1,497 @@
+        >
+          <s-button>
+            {{ word }}
+          </s-button>
+        </div>
+      </div>
+      <div class="login__text-confirm">
+        <p>
+          {{ t('desktop.confirmSeedText') }}
+        </p>
+      </div>
+      <div class="delimiter"></div>
+      <div class="login__correct-order">
+        <div
+          v-for="(word, idx) in seedPhraseToCompare"
+          @click="discardWord"
+          :key="'text' + Number(idx + 1)"
+          class="login__random-word"
+        >
+          <s-button>
+            {{ word }}
+          </s-button>
+        </div>
+      </div>
+      <div class="login__error">
+        <transition name="fade">
+          <span v-if="showErrorMessage" class="login__error-text">{{ t('desktop.errorMnemonicText') }}</span>
+        </transition>
+      </div>
+      <s-button
+        key="step2"
+        @click="handleMnemonicCheck"
+        class="s-typography-button--large login-btn"
+        :type="btnTypeConfirmStep"
+      >
+        {{ btnTextConfirmStep }}
+      </s-button>
+    </template>
+    <template v-else-if="step === LoginStep.CreateCredentials">
+      <div class="login__inputs">
+        <s-form>
+          <s-input
+            :placeholder="t('desktop.accountName.placeholder')"
+            v-model="accountName"
+            :disabled="loading"
+          ></s-input>
+          <p class="login__create-account-desc">{{ t('desktop.accountName.desc') }}</p>
+          <s-input
+            :type="inputType"
+            :placeholder="t('desktop.password.placeholder')"
+            v-model="accountPassword"
+            :disabled="loading"
+          >
+            <s-icon
+              :name="iconPasswordStyle"
+              size="18"
+              class="eye-icon"
+              slot="suffix"
+              @click.native="toggleVisibility"
+            />
+          </s-input>
+          <p class="login__create-account-desc">{{ t('desktop.password.desc') }}</p>
+          <s-input
+            type="password"
+            :placeholder="t('desktop.confirmPassword.placeholder')"
+            v-model="accountPasswordConfirm"
+            :disabled="loading"
+          ></s-input>
+        </s-form>
+      </div>
+      <div class="wallet-settings-create-token_export">
+        <s-switch v-model="toExport" :disabled="loading" />
+        <span>{{ t('desktop.exportOptionText') }}</span>
+      </div>
+      <p class="wallet-settings-create-token_desc">{{ t('desktop.exportJsonText') }}</p>
+      <a ref="json" class="download-json" />
+      <s-button
+        key="step3"
+        @click="createAccount"
+        class="s-typography-button--large login-btn"
+        type="primary"
+        :disabled="isInputsNotFilled"
+      >
+        {{ t('desktop.button.createAccount') }}
+      </s-button>
+    </template>
+  </div>
+</template>
+
+<script lang="ts">
+import { Mixins, Component, Prop, Ref } from 'vue-property-decorator';
+import { isEqual } from 'lodash';
```

Comment:
please do not use such import
from 'lodash';

import isEqual from 'lodash/fp/isEqual';